src/components/sky-nav/sky-nav.twig

```diff
@@ -72,6 +77,26 @@
     </nav>
   </header>
 </div>
+<script>
+  /**
+   * The Sky Nav is progressively enhanced with JS at smaller viewports.
+   * By default it is open for when JS is not available. In cases where JS
+   * _is_ available, this causes a poor loading UX where the menu jumps from
+   * open to closed causing a large layout shift.
+   *
+   * To keep the progressive enhancement in place _and_ not have the layout
+   * shift, the Sky Nav component assumes `no-js` to begin with. If JS is enabled:
+   *
+   * 1. Remove the `no-js` state (which removes no-JS menu styles)
+   * 2. Add the `is-loading` state. This happens before the Sky Nav JS has loaded
+   *    allowing us a hook to hide the menu items to remove the layout shift. Once
+   *    the Sky Nav JS loads, it removes the `is-loading` state CSS class and the
+   *    menu functions as was originally designed.
+   */
```

Comment:
Can we use a twig comment instead so it goes away at runtime?

Reply:
https://github.com/cloudfour/cloudfour.com-patterns/pull/1947/commits/4fd341d658d739707c44f8b1bb344616b07aedb0